benchmarks/grpo/grpo_local/run_engine.py

```
@@ -0,0 +1,78 @@
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--model-gguf", required=True, help="Path to GGUF model")
+    parser.add_argument("--tokenizer", required=True, help="Path to tokenizer.json")
+    parser.add_argument("--dataset", default="datasets/gsm8k_grpo.jsonl")
+    parser.add_argument("--output", default="results/grpo_local_gsm8k.json")
+    parser.add_argument("--steps", type=int, default=200)
+    args = parser.parse_args()
+
+    start_time = time.time()
+
+    # Build the Go binary if needed
+    build_cmd = ["go", "build", "-tags", "cgo,grpo_engine", "-o", "/tmp/grpo_bench",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/grpo_bench"

(S108)